src/app/misc-services/nonprofits.service.ts

```diff
@@ -1,7 +1,7 @@
 import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { Observable } from 'rxjs';
-import { baseOrganizationsUrl, baseOrganizationUrl } from '../utils/Constants';
+import { baseOrganizationsUrl, baseOrganizationUrl, searchOrganizationsUrl} from '../utils/Constants';
```

Comment:
[SUGGESTION] If you go back to the backend and look at the main getOrganizations endpoint, it supports search and searchType params. In an effort to avoid duplication and separation of the backend into too many endpoints, I think it would be best to just utilize the existing endpoint and baseOrganizationsUrl as opposed to the newer search endpoint. It shouldn't be too much to revert:

// api will return valid data at ~/api/organizations?parameter=<parameter>&parameter2=<parameter2>...etc
so this would look like ~/api/organizations?search=........

hopefully that works if not, then we can always use the searchorganizations endpoint